model-generation/environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - ipykernel
   - jupyter
   - nbconvert
+  - pandas
```

Comment:
Why did you add pandas again?

Reply:
it was needed by some other PR (the progres bar one)